domain/build/generated/source/buildConfig/debug/org/oppia/domain/BuildConfig.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Automatically generated file. DO NOT MODIFY
```

Comment:
Why do we need this file? I haven't seen it generated yet. If we don't need it, can we remove it?

Reply:
It was auto generated. Its been removed now